### PR TITLE
Add basic UI

### PR DESCRIPTION
**Pull request by ChrispaixK**
# OOP school library App 📚 
> Implement a basic User Interface

In this PR I :
* Create `app.rb`
  * Add `List all books` functionality
  * Add `List all people` functionality
  * Add `Create person` functionality
  * Add `Create rental` functionality
  * Add `List all rentals filtered by user id` functionality
 * Create `main.rb` with its functionalities as [required](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/school_library_ui.md)
 * Fix linters errors locally

### CODE REVIEW

File: app.rb [hunk 1]
```diff
@@ -0,0 +1,75 @@
+require_relative './student'
```
Comment (MahmoudBakr23) [inline app.rb:1]: HIGHLIGHTS

- Great job making your app file inherits data from your all other classes

File: main.rb [hunk 2]
```diff
@@ -0,0 +1,109 @@
+require_relative './app'
+
+def print_options
+  puts ''
+  puts 'Please choose and option by entering a number:'
```
Comment (MahmoudBakr23) [inline main.rb:5]: HIGHLIGHTS

- Well done managing the main file to handle the UI

**Review by MahmoudBakr23 — APPROVED** [review]
Hi @ChrispaixK 😃, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉

**Highlights**
- Well done handling the UI as the project requirements dictate
- App's working properly
- Descriptive PR summary
- Correct Linters

**Optional**
- Please fix the MIT link for a professional Readme file 

Cheers and Happy coding!👏👏👏

Feel free to mention @MahmoudBakr23 to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ChrispaixK** [comment]
Thanks a lot @MahmoudBakr23

**Pull request merged**